runtime/compiler/ilgen/Walker.cpp

```diff
@@ -6623,12 +6623,17 @@ TR_J9ByteCodeIlGenerator::genFlattenableWithField(uint16_t fieldCpIndex, TR_Opaq
 void
 TR_J9ByteCodeIlGenerator::genAconst_init(uint16_t cpIndex)
    {
-   TR_OpaqueClassBlock *valueTypeClass = method()->getClassFromConstantPool(comp(), cpIndex);
-   genAconst_init(valueTypeClass);
+   TR_OpaqueClassBlock *valueTypeClass = method()->getClassFromConstantPool(comp(), cpIndex, true /* returnClassForAot */);
+
+   if (comp()->getOption(TR_TraceILGen))
+      {
+      traceMsg(comp(), "%s: cpIndex %p valueTypeClass %p\n", __PRETTY_FUNCTION__, cpIndex, valueTypeClass);
```

Comment:
I'll update it to `__FUNCTION__`. Since there are two `genAconst_init` methods, I thought to use `__PRETTY_FUNCTION__` which prints the function signature to differentiate them.

Reply:
Addressed in a59bddb